### ISSUE TITLE
Any idea on how to apply gradient checkpointing on Efficientnet with FPN?

### ISSUE THREAD

**Issue by maxwxzheng** [qubvel/segmentation_models.pytorch#459]
Hi! I've been using Efficientnet with FPN on one of my projects. Recently there is a need to support bigger images with this architecture. I've been trying to add [checkpoint](https://pytorch.org/docs/stable/checkpoint.html) to the implementation but can't make it work. For all the different ways that I tried, the model trained with checkpointing performs significantly worse than without checkpointing. I think the biggest issues are the dropout_connect and batchnorms in the MBConvBlocks in Efficientnet.
Anyone has any idea on how to add gradient checkpointing to this architecture?
